samples/fabric-healthcare-data-solutions/README.md

```diff
@@ -0,0 +1,21 @@
+# Microsoft Fabric Healthcare Data Solutions Git Integration Helper
```

Comment:
"Healthcare data solutions in Microsoft Fabric ALM Helper"

Since it's not just for git it's also deployment pipelines.